### PR TITLE
Update of the analysis part to 76X

### PR DESCRIPTION
**Pull request by emanueledimarco**
- use of calibrated leptons
- useful scripts to transfer trees to another T2
- import some scripts for skimming (with few changes) from TTHAnalysis

### CODE REVIEW

File: MonoXAnalysis/scripts/copyTreesFromEOSToDcache.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    if raw_input()!='y':
+        print 'Aborting'
+        exit()
+
+    command = 'lcg-cp -b --vo cms -D srmv2 -U srmv2 -v'
+    eosconn='srm://srm-eoscms.cern.ch:8443/srm/v2/server?SFN='
+    remconn=''
+    if options.destT2=="T2_IT_Rome": remconn='srm://cmsrm-se01.roma1.infn.it:8443/srm/managerv2?SFN='
+    else:
+        print 'Only T2_IT_Rome implemented. Exiting.'
```
Comment (gpetruc) [inline MonoXAnalysis/scripts/copyTreesFromEOSToDcache.py]: You can use phedex to figure out that magic string for any T2.
E.g. if you do a query for `https://cmsweb.cern.ch/phedex/datasvc/json/prod/lfn2pfn?protocol=srm&node=T2_IT_Rome&lfn=/store/foo` you get a json which contains `"pfn":"srm://cmsrm-se01.roma1.infn.it:8443/srm/managerv2?SFN=/pnfs/roma1.infn.it/data/cms/store/foo"`.

File: MonoXAnalysis/cfg/run_monojet_cfg.py [hunk 2]
```diff
@@ -8,6 +8,8 @@
 # Load all analyzers
 from CMGTools.MonoXAnalysis.analyzers.dmCore_modules_cff import * 
 from PhysicsTools.HeppyCore.framework.heppy_loop import getHeppyOption
+from CMGTools.HToZZ4L.tools.configTools import * 
+
```
Comment (mariadalfonso) [inline MonoXAnalysis/cfg/run_monojet_cfg.py:12]: The AAA has been moved here, you need to change
https://github.com/CERN-PH-CMG/cmgtools-lite/pull/25

File: MonoXAnalysis/scripts/copyTreesFromEOSToDcache.py (hunk 1 above)
Comment (emanueledimarco) [inline MonoXAnalysis/scripts/copyTreesFromEOSToDcache.py]: done. Just laziness...

File: MonoXAnalysis/cfg/run_monojet_cfg.py (hunk 2 above)
Comment (emanueledimarco) [inline MonoXAnalysis/cfg/run_monojet_cfg.py:12]: @mariadalfonso done.

**Pull request merged**